### DISCUSSION TITLE
Why cannot using the same dataset run on different models?

### DISCUSSION THREAD

**Discussion by wuQi-666** [open-compass/opencompass#489] [Q&A]
Can the C-Eval dataset be used to run on the qwen-7b-Chat dataset, but there are no running results on Baichun2-13B-Chat?

![image](https://github.com/open-compass/opencompass/assets/139730426/173c6eed-f4e7-49c3-8c04-fcf8d1649121)
![image](https://github.com/open-compass/opencompass/assets/139730426/c0726959-1ea2-4fa0-b710-32032543aeb9)

对Baichuan2-13B-Chat的评测一直显示如上图所示，查看显卡内存也没有变化

**Comment by Leymore** [comment]
Please try to run Baichun2-13B-Chat with an addition `--debug` option to show more logs. 

You may also need to check your `transformers` version as Baichun2-13B-Chat cannot run properly with `transformers==4.34.0`

**Reply by wuQi-666** [reply]
Thank you for your answer. I followed the steps to create a virtual environment in openCompass and checked my own environment with transformers=4.34.0. May I ask how many versions of transformers should I choose?

**Reply by Leymore** [reply]
`transformers==4.33.3` works for me

**Reply by wuQi-666** [reply]
I have replaced transformers=4.33.3 and run the command CUDA_ VISIBLE_ DEVICES=2 Python run. py configs/eval_ Baihuan2. py -- debug, but the model reported an error as follows:
![image](https://github.com/open-compass/opencompass/assets/139730426/3e661204-800d-4f0e-b9cc-c7681fe67092)
The following is the file configs/eval_baihuan2. py  content 
![image](https://github.com/open-compass/opencompass/assets/139730426/4fa2a180-90b1-4256-b6f7-7e5f3ba2356d)

**Reply by Leymore** [reply]
`models = [*bc_models]` ?

**Reply by wuQi-666** [reply]
This is the code of hf_baichuan2_13b_chat.py 
![image](https://github.com/open-compass/opencompass/assets/139730426/fc931705-8e63-446f-92c1-f1c2a524ca46)

**Reply by wuQi-666** [reply]
In addition, debug using the command line, as follows: CUDA_VISIBLE_DEVICES=2 python run.py --models hf_baichuan2_13b_chat --datasets ceval_gen_5f30c7 --debug, the final run result is empty. 
![image](https://github.com/open-compass/opencompass/assets/139730426/0ceb66d7-23cb-431f-9885-d7677ad6a697)

Some of them reported errors, as shown in the following figure:
![image](https://github.com/open-compass/opencompass/assets/139730426/923b4b80-fd69-49ac-ace6-a7e6734c5682)

**Reply by Leymore** [reply]
The prediction is missing due to inference stage failure. I find this totally understandable.

**Reply by wuQi-666** [reply]
how can i fix it ?

**Reply by Leymore** [reply]
Please show the log in inference stage?

**Reply by wuQi-666** [reply]
I'm very sorry, the following error was caused by myself. When I just ran, the graphics card was not enough, which caused the above inference stage error. It is indeed possible to run correctly after changing the Transformers version. Thank you.
